cmd/check.go

```diff
@@ -468,6 +469,19 @@ func parseExportArg(arg string) (formatter controldisplay.Formatter, targetFileN
 	if formatter, found = controldisplay.GetDefinedExportFormatter(arg); found {
 		return
 	}
+
+	// no defined formatter
+	// HACK to resolve CSV and JSON filename exports
+	// We need this till we build out the templates for CSV and JSON
+	ext := filepath.Ext(arg)
+	if strings.HasSuffix(ext, "csv") {
```

Comment:
why not 
```
if ext == constants.CsvExtension 
```

and 
```
if ext == constants.JsonExtension && !strings.HasSuffix(arg, "asff.json")
```

Reply:
`ext == constants.CsvExtension` 
can't do this, since `golang` keeps the preceding `.` in the output of `filepath.Ext`